test/problems/divsort/generators/generators.yaml

```diff
@@ -2,7 +2,7 @@ solution: /submissions/accepted/th.py
 data:
   sample:
     data:
-      - "":
+      - '':
```

Comment:
Can we set `preserve_quotes = True` in the YAML reader/writer to avoid silly diffs like this?

(also, I think we should force double-quotes for YAML strings that BAPCtools creates, but that's probably out-of-scope here :stuck_out_tongue: )